db/schema.rb

```diff
@@ -36,8 +36,8 @@
   create_table "posts", force: :cascade do |t|
     t.string "title"
     t.string "comments"
-    t.integer "commentscounter"
-    t.integer "likescounter"
+    t.integer "commentscounter" default: 0
+    t.integer "likescounter" default: 0
```

Comment:
- [ ] It's giving error including line 52. It should be `t.integer "commentscounter", default: 0` like this but editing directly the schema is not a good practice. Whatever you need to change, change by creating a migration.